feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/staking/main/StakingViewState.kt

```diff
@@ -259,6 +262,11 @@ class WelcomeViewState(
         ReturnsModel(monthlyEstimation, yearlyEstimation)
     }.asLiveData(scope)
 
+    fun infoActionClicked() {
+        val reward = returns.value
+        reward?.let { _showRewardEstimationEvent.value = Event(StakingRewardEstimationBottomSheet.Payload(reward.monthly, reward.yearly)) }
```

Comment:
To calculate max apy, you should add `getMaxApy(): Double` method to  reward calculator, which should just take maximum value from `apyByValidator` mapping

Reply:
Done